src/background.ts

```diff
@@ -74,6 +74,10 @@ console.log(`Environment: ${import.meta.env.MODE}, appData: voicevox${suffix}`);
 
 // Electronの設定ファイルの保存場所を変更
 const beforeUserDataDir = app.getPath("userData"); // 設定ファイルのマイグレーション用
+
+//appnameをVoiceVoxとしてsetする
+app.setName(`voicevox${suffix}`);
+
 const fixedUserDataDir = path.join(app.getPath("appData"), `voicevox${suffix}`);
 if (!fs.existsSync(fixedUserDataDir)) {
   fs.mkdirSync(fixedUserDataDir);
```

Comment:
このあたりコメントが少しややこしくなっちゃっているので整理させてください！
こんな感じ･･･？

```suggestion

// バージョン0.14より前の設定ファイルの保存場所
const beforeUserDataDir = app.getPath("userData"); // マイグレーション用

// appnameをvoicevoxとしてsetする
app.setName(`voicevox${suffix}`);

// Electronの設定ファイルの保存場所を変更
const fixedUserDataDir = path.join(app.getPath("appData"), `voicevox${suffix}`);
if (!fs.existsSync(fixedUserDataDir)) {
  fs.mkdirSync(fixedUserDataDir);
```